### DISCUSSION TITLE
Evaluation time is too long

### DISCUSSION THREAD

**Discussion by havetry** [PaddlePaddle/PaddleSpatial#265] [BaiduKDDCup2022]
Dear organizer：
As the competition reaches the third stage, everyone has entered the sprint stage, and the evaluation time is too long. It is completely impossible to submit three times a day. Are you considering adding servers?

**Comment by ramber1836** [comment]
We are working on it and will let you know when the solution is figured out.

**Comment by ramber1836** [comment]
More GPU resources (+300%) will be deployed when Phase 3 starts. Hope this can make the scheduling much faster.

**Reply by Ethan1X** [reply]
Very nice, thanks.
Maybe not only GPU resources are needed, but CPU resources should also be increased.

**Reply by VEWOXIC** [reply]
We can finish the evaluation in about 3min on our machine. But it always meet timeout problem on the evaluation platform. Is it a common issue?

**Reply by VEWOXIC** [reply]
It seems that the code is not ran and just queued for 2 hours.

**Reply by ramber1836** [reply]
> Very nice, thanks. Maybe not only GPU resources are needed, but CPU resources should also be increased.

Yes, CPU resources had already been increased. The current main bottleneck is the GPU resource.

**Reply by zhanshijinwat** [reply]
> We can finish the evaluation in about 3min on our machine. But it always meet timeout problem on the evaluation platform. Is it a common issue?



> We can finish the evaluation in about 3min on our machine. But it always meet timeout problem on the evaluation platform. Is it a common issue?

If you cost 3min in your computer, I think you will timeout on the server. you only test one file in your computer, But there are many file s one the server. I cost 36s in my computer, and i also timeout on server( I need CPU not GPU). Three days before, the same file haven't timeout.

**Reply by aptx1231** [reply]
> > Very nice, thanks. Maybe not only GPU resources are needed, but CPU resources should also be increased.
> 
> Yes, CPU resources had already been increased. The current main bottleneck is the GPU resource.

However, I think that the test time is mainly consumed by constantly cycling through the test data, i.e. the bottleneck is the CPU. The reason is that there is only a small increase in time, not a multiplicative increase, using the five-fold model compared to using a single model.
